cmake/Dependencies.cmake

```diff
@@ -42,9 +42,6 @@ if (BUILD_NVJPEG)
   if (${NVJPEG_DECOUPLED_API})
     add_definitions(-DNVJPEG_DECOUPLED_API)
   endif()
-else()
```

Comment:
Why you are removing this statuses?

Reply:
With @klecki we figured out, it's gonna be better, to keep them in one place, `propagate_option` to be precise. I added it there in this PR also

Reply:
Yup, we now log the options when we propagate them.

Reply:
So all of them are listed.